scripts/add-validator/README.md

```diff
@@ -0,0 +1,38 @@
+# Add Validator
+
+This script will help us in adding the validator node to blockchain programmaticaly. 
+
+<br />
+
+## Requirements
+[NodeJs (v14.17.0 lts)](https://nodejs.org/en/download/ "NodeJs (v14.17.0 lts)")
```

Comment:
Where do we use it?

Reply:
It will be useful if we are executing the scripts in the local machine without docker.

Reply:
We need to request the same version which is in the Dockerfile.

Reply:
Updated it, with docker file